catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/AbstractCswSource.java

```diff
@@ -1018,6 +1018,7 @@ private void alignStream(InputStream in, long requestedBytesToSkip, long respons
         throw new IOException("Server skipped more bytes than requested in the range header.");
       }
     } catch (IOException e) {
+      in.close();
```

Comment:
✏️ I think `IOUtils.closeQuietly` would be more appropriate here so you don't throw an exception from here and fail to generate the exception you want.

Reply:
Good call, will update.